content/en/blog/2022/meet-our-contributors-APAC-China-region-03.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: blog
+title: "Meet Our Contributors - APAC (China region)"
+date: 2022-06-15
+slug: meet-our-contributors-chn-ep-03
+---
+
+**Authors & Interviewers:** [Avinesh Tripathi](https://github.com/AvineshTripathi), [Debabrata Panigrahi](https://github.com/Debanitrkl), [Jayesh Srivastava](https://github.com/jayesh-srivastava), [Priyanka Saggu](https://github.com/Priyankasaggu11929/), [Purneswar Prasad](https://github.com/PurneswarPrasad), [Vedant Kakde](https://github.com/vedant-kakde)
+
+---
+
+Hello, everyone 👋
+
+Welcome back to the third episode of the "Meet Our Contributors" blog post series for APAC.
+
+This post features four outstanding contributors from  China, who have played diverse leadership and community roles in the upstream Kubernetes project.
+
+So, without further ado, let's get straight to the article.
+
+
+## [Andy Zhang](https://github.com/andyzhangx)
+
+Andy Zhang currently works for Microsoft China at the Shanghai site. His main focus is on Kubernetes storage drivers. Andy started contributing to Kubernetes about 5 years ago.
+
+He states that as he is working in Azure Kubernetes Service team and spends most of his time contributing to the Kubernetes community project. Now he is the main contributor of quite a lot Kubernetes subprojects such as Kubernetes cloud provider code.
+
+His open source contributions are mainly self-motivated. In the last two years he has mentored a few students contributing to Kubernetes through the LFX Mentorship program, some of whom got jobs due to their expertise and contributions on Kubernetes projects.
+
+Andy is an active member of the China Kubernetes community. He adds that the Kubernetes community has a good guide about how to become members, code reviewers, approvers and finally when he found out that some open source projects are in the very early stage, he actively contributed to those projects and became the project maintainer.
+
+
+## [Shiming Zhang](https://github.com/wzshiming)
+
+Shiming Zhang is a Software Engineer with DaoCloud in Shanghai, China. 
+
+He has mostly been involved with SIG Node, as a reviewer.
+
+As of his prior experience, Shiming had been involved with Kubernetes at his company. His major contributions have mainly been bug fixes and feature improvements in an ongoing [KEP](https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/2712-pod-priority-based-graceful-node-shutdown), all revolving around SIG Node.
```

Comment:
```suggestion
Shiming Zhang is a Software Engineer working on Kubernetes for DaoCloud in Shanghai, China.

He has mostly been involved with SIG Node, as a reviewer. His major contributions have mainly been bug fixes and feature improvements in an ongoing [KEP](https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/2712-pod-priority-based-graceful-node-shutdown), all revolving around SIG Node.
```

Reply:
This is my suggested re-write to eliminate the problematic sentence.

Reply:
@jberkus Done!

Reply:
@jberkus Done!